pkg/transfers/inbound/cleanup.go

```diff
@@ -25,13 +28,34 @@ func Cleanup(logger log.Logger, agent upload.Agent, dl *downloadedFiles) error {
 	if err := deleteFilesOnRemote(logger, agent, dl.dir, agent.ReturnPath()); err != nil {
 		el.Add(err)
 	}
+	if el.Empty() {
+		return nil
+	}
+	return el
+}
+
+// CleanupEmptyFiles deletes empty ACH files if file is older than value in config
+func CleanupEmptyFiles(logger log.Logger, agent upload.Agent, dl *downloadedFiles, now time.Time, after time.Duration) error {
+	var el base.ErrorList
+
+	if after <= 0*time.Second {
+		logger.Log(fmt.Sprintf("deleting empty file requires after > 0. currently: %s", after))
```

Comment:
You could use `Logf()` here and in a couple of other places.

Reply:
Totally! Pushed.